File: src/shared/py/langboard_shared/domain/models/BotScheduleGraphApprovalRequest.py
from .bases import BaseGraphApprovalBotRequest
from .GraphApprovalRequest import GraphApprovalOriginType


class BotScheduleGraphApprovalRequest(BaseGraphApprovalBotRequest, table=True):
    @staticmethod
    def get_request_type() -> GraphApprovalOriginType:
        return GraphApprovalOriginType.Schedule

File: src/socket/src/models/BotScheduleGraphApprovalRequest.ts
import BaseGraphApprovalBotRequest from "@/models/bases/BaseGraphApprovalBotRequest";
import { Entity } from "typeorm";

@Entity({ name: "bot_schedule_graph_approval_request" })
class BotScheduleGraphApprovalRequest extends BaseGraphApprovalBotRequest {}

export default BotScheduleGraphApprovalRequest;
